Build.PL

```diff
@@ -52,13 +53,16 @@ my $builder = $class->new(
           'MooseX::AttributeCloner'      => 0.17,
           'MooseX::Getopt'               => 0,
           'MooseX::StrictConstructor'    => 0,
+          'Net::AMQP::RabbitMQ'          => 0,
```

Comment:
What are prerequisites of Net::AMQP::RabbitMQ for deployment purposes?

Reply:
Think we'll need the `librabbitmq-dev` and `uuid-dev` packages installed.